RELEASE_PROCESS.md

```diff
@@ -297,3 +317,13 @@ spelled with complete minutiae.
       1. A blue box will appear above the job list with a right side drop
          -down. Click the drop-down and specify the version number in the
          dialog that appears
+
+1. Annouce the release
+      1. For major and minor releases, write a blog and publish it to blog.podman.io
```

Comment:
```suggestion
      1. For major and minor releases, write a blog and publish it to blogs.podman.io
```
Double check with @bbaude, but I think we've been using the plural blogs in our documentation/presentations.  Either works.